src/pages/Lend/LoanChannelCategoryPage.vue

```diff
@@ -301,7 +330,9 @@ export default {
 		next();
 	},
 	beforeRouteLeave(to, from, next) {
-		if (typeof window !== 'undefined' && to.path.indexOf('/lend/') !== -1) {
+		if (typeof window !== 'undefined'
+			&& to.path.indexOf('/lend/') !== -1
+			&& to.path.indexOf('/lend/filter') === -1) {
```

Comment:
new check so we DO NOT set this cookie if they are going to /lend/filter. Only when directed to legacy /lend/some-cat.